components/automate-cli/cmd/chef-automate/version.go

```diff
@@ -43,11 +44,10 @@ func printClientVersion() {
 	if verbose {
 		writer.Println("CLI")
 		writer.Println("===")
-		writer.Printf("Version: %s\n", version.BuildTime)
-		writer.Printf("Build time: %s\n", version.BuildTime)
+		writer.Printf("CLI Build: %s\n", version.BuildTime)
 		writer.Printf("Git SHA: %s\n", version.GitSHA)
 	} else {
-		writer.Printf("CLI version: %s\n", version.BuildTime)
+		writer.Printf("CLI Build: %s\n", version.BuildTime)
```

Comment:
Super nitpicky, but `Println` can be used when you want the thing to appear at the end of the line, like `writer.Println("CLI Build:", version.BuildTime)`. 

https://play.golang.org/p/XxFm4WS-mNg

Reply:
Unfortunately, it looks like the api of that writer object doesn't exactly matching the api of fmt and it only accepts a single string.  We should probably make them match to avoid confusion.